src/ontology/mondo-edit.obo

```diff
@@ -411074,6 +411074,7 @@ name: Alice in Wonderland syndrome
 def: "A disorienting neuropsychological condition that affects perception. People experience size distortion such as micropsia, macropsia, pelopsia, or teleopsia. Size distortion may occur of other sensory modalities." [Wikipedia:Alice_in_Wonderland_syndrome]
 xref: MESH:D062026 {source="MONDO:equivalentTo"}
 xref: NCIT:C116362 {source="MONDO:equivalentTo"}
+is_a: MONDO:0002025 {source="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC4909520/"} ! psychiatric disorder
```

Comment:
can you add the source as PMID:XXXXXXXX
(the PMID should be on the PMC article)